Automattic-Tracks-iOS.podspec

```diff
@@ -1,36 +1,42 @@
-Pod::Spec.new do |spec|
-  spec.name         = 'Automattic-Tracks-iOS'
-  spec.version      = File.read("Automattic-Tracks-iOS/TracksConstants.m").split("const TracksLibraryVersion = @\"").last.split("\"").first
-  spec.license      = { :type => 'GPLv2' }
-  spec.homepage     = 'https://github.com/automattic/automattic-tracks-ios'
-  spec.authors      = { 'Automattic' => 'mobile@automattic.com' }
-  spec.summary      = 'Simple way to track events in an iOS app with Automattic Tracks internal service'
-  spec.source       = { :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :tag => spec.version.to_s }
-  spec.swift_version = '5.0'
-
-  spec.ios.source_files = 'Automattic-Tracks-iOS/**/*.{h,m,swift}'
-  spec.ios.exclude_files = 'Automattic-Tracks-OSX/Automattic_Tracks_OSX.h'
-
-  spec.osx.source_files = 'Automattic-Tracks-iOS/**/*.{h,m,swift}'
-  spec.osx.exclude_files = ['Automattic-Tracks-iOS/Automattic-Tracks-iOS.h', 'Automattic-Tracks-iOS/ABTesting/*']
-
-  spec.private_header_files = 'Automattic-Tracks-iOS/Internal Logging/TracksLoggingPrivate.h'
-  spec.resource_bundle = { 'DataModel' => ['Automattic-Tracks-iOS/**/*.xcdatamodeld'] }
-
-  spec.framework        = 'CoreData'
-  spec.ios.framework    = 'UIKit'
-  spec.ios.framework    = 'CoreTelephony'
-  spec.osx.framework    = 'AppKit'
-
-  spec.ios.deployment_target  = '12.0'
-  spec.osx.deployment_target  = '10.12'
-
-  spec.header_dir = 'AutomatticTracks'
-  spec.module_name = 'AutomatticTracks'
-
-  spec.ios.dependency 'UIDeviceIdentifier', '~> 1'
-  spec.dependency 'CocoaLumberjack', '~> 3'
-  spec.dependency 'Reachability', '~> 3'
-  spec.dependency 'Sentry', '~> 6'
-  spec.dependency 'Sodium', '>= 0.9.1'
+Pod::Spec.new do |s|
+  s.name          = 'Automattic-Tracks-iOS'
+  s.version       = File.read("Automattic-Tracks-iOS/TracksConstants.m").split("const TracksLibraryVersion = @\"").last.split("\"").first
```

Comment:
I don't love the fact that this is as non-standard as it is (it's the only pod that works this way). WDYT about aligning it with the other projects?

Reply:
I actually pondered about that and would love to migrate to hardcoded value here indeed.

Thing is, if we do this it could be nice to also add in the CI a step that checks that the value here matches the one in the .m to ensure they are still always in sync and we do to forget to update both at once. Will try to tackle this next week.

Reply:
@jkmassel Fix in 2ea0fa57de1dd6c022fa60c81390802dd7d13675. See also [this comment below](https://github.com/Automattic/Automattic-Tracks-iOS/pull/175?xs#issuecomment-809268052) for details.